src/main/java/com/artipie/asto/rx/RxCopy.java

```diff
@@ -40,6 +40,11 @@
 @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
 public class RxCopy {
 
+    /**
+     * The default parallelism level.
+     */
+    private static final Integer DEFLT_PARALLELISM = 8;
```

Comment:
@Sammers21 I would be more correct to use `Runtime.getRuntime().availableProcessors()` (like in `ParallelFlowable`)